apps/condo/lang/en/en.json

```
@@ -1344,6 +1344,10 @@
   "notification.messages.METER_VERIFICATION_DATE_REMINDER.push.title": "Doma ⏰",
   "notification.messages.METER_SUBMIT_READINGS_REMINDER.email.subject": "Let's submit meter readings?",
   "notification.messages.METER_SUBMIT_READINGS_REMINDER.push.title": "Doma ⏰",
+  "notification.messages.METER_SUBMIT_READINGS_REMINDER_START_PERIOD.email.subject": "It's time to submit your meter readings! The period for submitting meter readings for {data.monthName} has begun",
+  "notification.messages.METER_SUBMIT_READINGS_REMINDER_START_PERIOD.push.title": "Remind ⏰",
+  "notification.messages.METER_SUBMIT_READINGS_REMINDER_END_PERIOD_TYPE.email.subject": "Submit your meter readings! Today is the last day to submit meter readings for {data.monthName}.",
+  "notification.messages.METER_SUBMIT_READINGS_REMINDER_END_PERIOD_TYPE.push.title": "Remind ⏰",
```

Maybe as a noun, not a verb?

```suggestion
  "notification.messages.METER_SUBMIT_READINGS_REMINDER_END_PERIOD_TYPE.push.title": "Reminder ⏰",
```

Why not to say directly "Submit your meter readings!"?

---

this is a text from product